src/RestClient.Net.UnitTests/Snippets.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ApiExamples.Model.JsonModel;
+using RestClient.Net.Samples.Model;
+using RestClient.Net.UnitTests.Model;
+
+namespace RestClient.Net.UnitTests
+{
+    [TestClass]
```

Comment:
These are going to make actual http calls which is no good because they can't be reliable.

These would need to be excluded from the pipeline